package.json

```diff
@@ -2,9 +2,12 @@
   "name": "core-decorators",
   "version": "0.0.13",
   "description": "Library of ES7 decorators inspired by languages that come with built-ins like @override, @deprecated, etc",
-  "main": "src/core-decorators.js",
+  "main": "lib/core-decorators.js",
+  "files": "lib",
```

Comment:
I'm having a hard time deciding on this one. I and several others are indeed using this via npm right now as-is, stage 0, ES7+ (fed into babel obviously). Debugging and source maps being my first concern (can be overcome), second being that people will also certainly "want" distributed builds in their preferred target module format (CJS/AMD/Globals/whatever).

Let me think on this for a bit. Feel free to chime in with your thoughts as well.